### ISSUE TITLE
AbemaTV Broken, unable to parse page

### ISSUE THREAD

**Issue by Adthrawn** [yt-dlp/yt-dlp#7244]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I'm running yt-dlp version **2023.03.04** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Japan

### Provide a description that is worded well enough to be understood

I am attempting to download a VOD/PPV from AbemaTV, but am getting a 404 error. I have also tried to download using the m3u8 directly and it will fail as well. I've included the verbose output from the url attempt into the box below. I'll add the m3u8 output as well in subsequent comments

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['--cookies-from-browser', 'firefox', 'https://abema.tv/channels/payperview-5/slots/9uYsUEkm87qk6K', '-vU']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version nightly@2023.06.04.142854 [c2a1bdb00] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.22631-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg 2022-06-16-git-5242ede48d-essentials_build-www.gyan.dev (setts), ffprobe 2022-06-16-git-5242ede48d-essentials_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.18.0, brotli-1.0.9, certifi-2023.05.07, mutagen-1.46.0, sqlite3-2.6.0, websockets-11.0.3
[Cookies] Extracting cookies from firefox
[debug] Extracting cookies from: "C:\Users\mseli\AppData\Roaming\Mozilla\Firefox\Profiles\16yhddr1.default-beta\cookies.sqlite"
[Cookies] Extracted 2105 cookies from firefox
[debug] Proxy map: {}
[debug] Loaded 1840 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp-nightly-builds/releases/latest
Available version: nightly@2023.06.04.142854, Current version: nightly@2023.06.04.142854
Current Build Hash: 1ac1d528750b76c9b9e2f931791d35fd2e9f6f53586d40ac715bfda0471f7c40
yt-dlp is up to date (nightly@2023.06.04.142854)
[AbemaTV] Extracting URL: https://abema.tv/channels/payperview-5/slots/9uYsUEkm87qk6K
[AbemaTV] Authorizing
[AbemaTV] 9uYsUEkm87qk6K: Downloading webpage
[AbemaTV] 9uYsUEkm87qk6K: Checking playability
ERROR: [AbemaTV] 9uYsUEkm87qk6K: Unable to download JSON metadata: HTTP Error 404: Not Found (caused by <HTTPError 404: 'Not Found'>); please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
  File "yt_dlp\extractor\common.py", line 703, in extract
  File "yt_dlp\extractor\abematv.py", line 452, in _real_extract
  File "yt_dlp\extractor\common.py", line 1057, in download_content
  File "yt_dlp\extractor\common.py", line 1021, in download_handle
  File "yt_dlp\extractor\common.py", line 891, in _download_webpage_handle
  File "yt_dlp\extractor\common.py", line 848, in _request_webpage

  File "yt_dlp\extractor\common.py", line 830, in _request_webpage
  File "yt_dlp\YoutubeDL.py", line 3776, in urlopen
  File "urllib\request.py", line 531, in open
  File "urllib\request.py", line 640, in http_response
  File "urllib\request.py", line 569, in error
  File "urllib\request.py", line 502, in _call_chain
  File "urllib\request.py", line 649, in http_error_default
urllib.error.HTTPError: HTTP Error 404: Not Found
```

**Comment by Adthrawn** [comment]
```[debug] Command-line config: ['--cookies-from-browser', 'firefox', 'https://vod-abematv.akamaized.net/ppv/chase/slot/9uYsUEkm87qk6K/playlist.m3u8?t=DimKDQ77oyRRB91S5mu2dES2PL38RmuBFgRyksAzn3Z8F2jwfqdVkvggK2Ke94MsBbKYsXN9Wmf8Lwr87g8ZokFc1q2wDaduWzEWPSpBMfvvBWds2TKNd7HqXLRJJ4zPT4&enc=clear&dt=pc_firefox&ccf=0&ut=1&kg=2161', '-vU']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version nightly@2023.06.04.142854 [c2a1bdb00] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.22631-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg 2022-06-16-git-5242ede48d-essentials_build-www.gyan.dev (setts), ffprobe 2022-06-16-git-5242ede48d-essentials_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.18.0, brotli-1.0.9, certifi-2023.05.07, mutagen-1.46.0, sqlite3-2.6.0, websockets-11.0.3
[Cookies] Extracting cookies from firefox
[debug] Extracting cookies from: "C:\Users\mseli\AppData\Roaming\Mozilla\Firefox\Profiles\16yhddr1.default-beta\cookies.sqlite"
[Cookies] Extracted 2104 cookies from firefox
[debug] Proxy map: {}
[debug] Loaded 1840 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp-nightly-builds/releases/latest
Available version: nightly@2023.06.04.142854, Current version: nightly@2023.06.04.142854
Current Build Hash: 1ac1d528750b76c9b9e2f931791d35fd2e9f6f53586d40ac715bfda0471f7c40
yt-dlp is up to date (nightly@2023.06.04.142854)
[generic] Extracting URL: https://vod-abematv.akamaized.net/ppv/chase/slot/9uYsUEkm87qk6K/playlist.m3u8?t=DimKDQ77oyRRB91S5mu2dES2PL38RmuBFgRyksAzn3Z8F2jwfqdVkvggK2Ke94MsBbKYsXN9Wmf8Lwr87g8ZokFc1q2wDaduWzEWPSpBMfvvBWds2TKNd7HqXLRJJ4zPT4&enc=clear&dt=pc_firefox&ccf=0&ut=1&kg=2161
[generic] playlist: Downloading webpage
[debug] Identified a direct video link
[generic] playlist: Downloading m3u8 information
[generic] playlist: Checking m3u8 live status
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[debug] Default format spec: best/bestvideo+bestaudio
[info] playlist: Downloading 1 format(s): 8300
[debug] Invoking ffmpeg downloader on "https://vod-abematv.akamaized.net/ppv/chase/slot/9uYsUEkm87qk6K/1080/playlist.m3u8?aver=AXmEPYkegEgNTZ&ccf=0&dt=pc_firefox&enc=clear"
[download] Destination: playlist 2023-06-05 11_22 [playlist].mp4
[debug] ffmpeg command line: ffmpeg -y -loglevel verbose -headers "User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
Accept-Language: en-us,en;q=0.5
Sec-Fetch-Mode: navigate
" -i "https://vod-abematv.akamaized.net/ppv/chase/slot/9uYsUEkm87qk6K/1080/playlist.m3u8?aver=AXmEPYkegEgNTZ&ccf=0&dt=pc_firefox&enc=clear" -c copy -f mpegts "file:playlist 2023-06-05 11_22 [playlist].mp4.part"
ffmpeg version 2022-06-16-git-5242ede48d-essentials_build-www.gyan.dev Copyright (c) 2000-2022 the FFmpeg developers
  built with gcc 11.3.0 (Rev1, Built by MSYS2 project)
  configuration: --enable-gpl --enable-version3 --enable-static --disable-w32threads --disable-autodetect --enable-fontconfig --enable-iconv --enable-gnutls --enable-libxml2 --enable-gmp --enable-bzlib --enable-lzma --enable-zlib --enable-libsrt --enable-libssh --enable-libzmq --enable-avisynth --enable-sdl2 --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxvid --enable-libaom --enable-libopenjpeg --enable-libvpx --enable-mediafoundation --enable-libass --enable-libfreetype --enable-libfribidi --enable-libvidstab --enable-libvmaf --enable-libzimg --enable-amf --enable-cuda-llvm --enable-cuvid --enable-ffnvcodec --enable-nvdec --enable-nvenc --enable-d3d11va --enable-dxva2 --enable-libmfx --enable-libgme --enable-libopenmpt --enable-libopencore-amrwb --enable-libmp3lame --enable-libtheora --enable-libvo-amrwbenc --enable-libgsm --enable-libopencore-amrnb --enable-libopus --enable-libspeex --enable-libvorbis --enable-librubberband
  libavutil      57. 27.100 / 57. 27.100
  libavcodec     59. 33.100 / 59. 33.100
  libavformat    59. 25.100 / 59. 25.100
  libavdevice    59.  6.100 / 59.  6.100
  libavfilter     8. 41.100 /  8. 41.100
  libswscale      6.  6.100 /  6.  6.100
  libswresample   4.  6.100 /  4.  6.100
  libpostproc    56.  5.100 / 56.  5.100
[tcp @ 000002114325f240] Starting connection attempt to 198.18.0.62 port 443
[tcp @ 000002114325f240] Successfully connected to 198.18.0.62 port 443
[hls @ 000002114325b780] Skip ('#EXT-X-VERSION:3')
[hls @ 000002114325b780] Skip ('#EXT-X-DISCONTINUITY-SEQUENCE:0')
[hls @ 000002114325b780] Skip ('#EXT-X-DISCONTINUITY')
[hls @ 000002114325b780] HLS request for url 'https://vod-abematv.akamaized.net/tspglive/payperview-5/9uYsUEkm87qk6K/527-580_s2_p100/h264/1080/K4rTJfiiGrEhAvpaoCP8zo.ts', offset 0, playlist 0
[hls @ 000002114325b780] Unable to open key file abematv-license://GA8BiJqHu5LzngNk5MujFoszeDtdHXbQ8Vu61McmM2zr
[hls @ 000002114325b780] Opening 'crypto+https://vod-abematv.akamaized.net/tspglive/payperview-5/9uYsUEkm87qk6K/527-580_s2_p100/h264/1080/K4rTJfiiGrEhAvpaoCP8zo.ts' for reading
[tcp @ 000002114325fe00] Starting connection attempt to 198.18.0.62 port 443
[tcp @ 000002114325fe00] Successfully connected to 198.18.0.62 port 443
[hls @ 000002114325b780] Error when loading first segment 'https://vod-abematv.akamaized.net/tspglive/payperview-5/9uYsUEkm87qk6K/527-580_s2_p100/h264/1080/Rs5LVsnUiFQSAeCCU7Q83W.ts'
[AVIOContext @ 0000021143805880] Statistics: 1049088 bytes read, 0 seeks
[AVIOContext @ 00000211433291c0] Statistics: 85291 bytes read, 0 seeks
https://vod-abematv.akamaized.net/ppv/chase/slot/9uYsUEkm87qk6K/1080/playlist.m3u8?aver=AXmEPYkegEgNTZ&ccf=0&dt=pc_firefox&enc=clear: Invalid data found when processing input


ERROR: ffmpeg exited with code 1
  File "yt_dlp\__main__.py", line 17, in <module>
  File "yt_dlp\__init__.py", line 984, in main
  File "yt_dlp\__init__.py", line 974, in _real_main
  File "yt_dlp\YoutubeDL.py", line 3402, in download
  File "yt_dlp\YoutubeDL.py", line 3377, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1541, in extract_info
  File "yt_dlp\YoutubeDL.py", line 1552, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1649, in __extract_info
  File "yt_dlp\YoutubeDL.py", line 1708, in process_ie_result
  File "yt_dlp\YoutubeDL.py", line 2815, in process_video_result
  File "yt_dlp\YoutubeDL.py", line 3280, in process_info
  File "yt_dlp\YoutubeDL.py", line 3003, in dl
  File "yt_dlp\downloader\common.py", line 456, in download
  File "yt_dlp\downloader\external.py", line 75, in real_download
  File "yt_dlp\YoutubeDL.py", line 1035, in report_error
  File "yt_dlp\YoutubeDL.py", line 964, in trouble```

**Comment by Adthrawn** [comment]
```[debug] Command-line config: ['--cookies-from-browser', 'firefox', 'https://vod-abematv.akamaized.net/ppv/chase/slot/9uYsUEkm87qk6K/180/playlist.m3u8?aver=AXmEPYkegEgNTZ&ccf=0&dt=pc_firefox&enc=clear&kg=2161', '-vU']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version nightly@2023.06.04.142854 [c2a1bdb00] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.22631-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg 2022-06-16-git-5242ede48d-essentials_build-www.gyan.dev (setts), ffprobe 2022-06-16-git-5242ede48d-essentials_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.18.0, brotli-1.0.9, certifi-2023.05.07, mutagen-1.46.0, sqlite3-2.6.0, websockets-11.0.3
[Cookies] Extracting cookies from firefox
[debug] Extracting cookies from: "C:\Users\mseli\AppData\Roaming\Mozilla\Firefox\Profiles\16yhddr1.default-beta\cookies.sqlite"
[Cookies] Extracted 2104 cookies from firefox
[debug] Proxy map: {}
[debug] Loaded 1840 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp-nightly-builds/releases/latest
Available version: nightly@2023.06.04.142854, Current version: nightly@2023.06.04.142854
Current Build Hash: 1ac1d528750b76c9b9e2f931791d35fd2e9f6f53586d40ac715bfda0471f7c40
yt-dlp is up to date (nightly@2023.06.04.142854)
[generic] Extracting URL: https://vod-abematv.akamaized.net/ppv/chase/slot/9uYsUEkm87qk6K/180/playlist.m3u8?aver=AXmEPYkegEgNTZ&ccf=0&dt=pc_firefox&enc=clear&kg=2161
[generic] playlist: Downloading webpage
[debug] Identified a direct video link
[generic] playlist: Downloading m3u8 information
[generic] playlist: Checking m3u8 live status
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[debug] Default format spec: best/bestvideo+bestaudio
[info] playlist: Downloading 1 format(s): 0
[debug] Invoking ffmpeg downloader on "https://vod-abematv.akamaized.net/ppv/chase/slot/9uYsUEkm87qk6K/180/playlist.m3u8?aver=AXmEPYkegEgNTZ&ccf=0&dt=pc_firefox&enc=clear&kg=2161"
[download] Destination: playlist 2023-06-05 11_24 [playlist].mp4
[debug] ffmpeg command line: ffmpeg -y -loglevel verbose -headers "User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
Accept-Language: en-us,en;q=0.5
Sec-Fetch-Mode: navigate
" -i "https://vod-abematv.akamaized.net/ppv/chase/slot/9uYsUEkm87qk6K/180/playlist.m3u8?aver=AXmEPYkegEgNTZ&ccf=0&dt=pc_firefox&enc=clear&kg=2161" -c copy -f mpegts "file:playlist 2023-06-05 11_24 [playlist].mp4.part"
ffmpeg version 2022-06-16-git-5242ede48d-essentials_build-www.gyan.dev Copyright (c) 2000-2022 the FFmpeg developers
  built with gcc 11.3.0 (Rev1, Built by MSYS2 project)
  configuration: --enable-gpl --enable-version3 --enable-static --disable-w32threads --disable-autodetect --enable-fontconfig --enable-iconv --enable-gnutls --enable-libxml2 --enable-gmp --enable-bzlib --enable-lzma --enable-zlib --enable-libsrt --enable-libssh --enable-libzmq --enable-avisynth --enable-sdl2 --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxvid --enable-libaom --enable-libopenjpeg --enable-libvpx --enable-mediafoundation --enable-libass --enable-libfreetype --enable-libfribidi --enable-libvidstab --enable-libvmaf --enable-libzimg --enable-amf --enable-cuda-llvm --enable-cuvid --enable-ffnvcodec --enable-nvdec --enable-nvenc --enable-d3d11va --enable-dxva2 --enable-libmfx --enable-libgme --enable-libopenmpt --enable-libopencore-amrwb --enable-libmp3lame --enable-libtheora --enable-libvo-amrwbenc --enable-libgsm --enable-libopencore-amrnb --enable-libopus --enable-libspeex --enable-libvorbis --enable-librubberband
  libavutil      57. 27.100 / 57. 27.100
  libavcodec     59. 33.100 / 59. 33.100
  libavformat    59. 25.100 / 59. 25.100
  libavdevice    59.  6.100 / 59.  6.100
  libavfilter     8. 41.100 /  8. 41.100
  libswscale      6.  6.100 /  6.  6.100
  libswresample   4.  6.100 /  4.  6.100
  libpostproc    56.  5.100 / 56.  5.100
[tcp @ 000002271cc6f180] Starting connection attempt to 198.18.0.62 port 443
[tcp @ 000002271cc6f180] Successfully connected to 198.18.0.62 port 443
[hls @ 000002271cc6b700] Skip ('#EXT-X-VERSION:3')
[hls @ 000002271cc6b700] Skip ('#EXT-X-DISCONTINUITY-SEQUENCE:0')
[hls @ 000002271cc6b700] Skip ('#EXT-X-DISCONTINUITY')
[hls @ 000002271cc6b700] HLS request for url 'https://vod-abematv.akamaized.net/tspglive/payperview-5/9uYsUEkm87qk6K/527-580_s2_p100/h264/180/2GnWcKMeC6GARvARdvwzFZ.ts', offset 0, playlist 0
[hls @ 000002271cc6b700] Unable to open key file abematv-license://GA8BiJqHu5LzngNk5MujFoszeDtdHXbQ8Vu61McmM2zr
[hls @ 000002271cc6b700] Opening 'crypto+https://vod-abematv.akamaized.net/tspglive/payperview-5/9uYsUEkm87qk6K/527-580_s2_p100/h264/180/2GnWcKMeC6GARvARdvwzFZ.ts' for reading
[tcp @ 000002271d1485c0] Starting connection attempt to 198.18.0.62 port 443
[tcp @ 000002271d1485c0] Successfully connected to 198.18.0.62 port 443
[AVIOContext @ 000002271d0ff100] Statistics: 70696 bytes read, 0 seeks
[hls @ 000002271cc6b700] HLS request for url 'https://vod-abematv.akamaized.net/tspglive/payperview-5/9uYsUEkm87qk6K/527-580_s2_p100/h264/180/2GnWcKMeC6Foq47zLgYGuV.ts', offset 0, playlist 0
[hls @ 000002271cc6b700] Opening 'crypto+https://vod-abematv.akamaized.net/tspglive/payperview-5/9uYsUEkm87qk6K/527-580_s2_p100/h264/180/2GnWcKMeC6Foq47zLgYGuV.ts' for reading
[tcp @ 000002271d1b1600] Starting connection attempt to 198.18.0.62 port 443
[tcp @ 000002271d1b1600] Successfully connected to 198.18.0.62 port 443
[AVIOContext @ 000002271cd3ee40] Statistics: 151712 bytes read, 0 seeks
[tcp @ 000002271d1b1600] Starting connection attempt to 198.18.0.62 port 443
[tcp @ 000002271d1b1600] Successfully connected to 198.18.0.62 port 443
[hls @ 000002271cc6b700] Skip ('#EXT-X-VERSION:3')
[hls @ 000002271cc6b700] Skip ('#EXT-X-DISCONTINUITY-SEQUENCE:0')
[hls @ 000002271cc6b700] Skip ('#EXT-X-DISCONTINUITY')
[hls @ 000002271cc6b700] HLS request for url 'https://vod-abematv.akamaized.net/tspglive/payperview-5/9uYsUEkm87qk6K/527-580_s2_p100/h264/180/2GnWcKMeC6FsgCqedNrj8u.ts', offset 0, playlist 0
[hls @ 000002271cc6b700] Opening 'crypto+https://vod-abematv.akamaized.net/tspglive/payperview-5/9uYsUEkm87qk6K/527-580_s2_p100/h264/180/2GnWcKMeC6FsgCqedNrj8u.ts' for reading
[tcp @ 000002271cd36c40] Starting connection attempt to 198.18.0.62 port 443
[tcp @ 000002271cd36c40] Successfully connected to 198.18.0.62 port 443
[AVIOContext @ 000002271d3c3140] Statistics: 187424 bytes read, 0 seeks
[hls @ 000002271cc6b700] HLS request for url 'https://vod-abematv.akamaized.net/tspglive/payperview-5/9uYsUEkm87qk6K/527-580_s2_p100/h264/180/2GnWcKMeC6FogmQf5UrCoC.ts', offset 0, playlist 0
[hls @ 000002271cc6b700] Opening 'crypto+https://vod-abematv.akamaized.net/tspglive/payperview-5/9uYsUEkm87qk6K/527-580_s2_p100/h264/180/2GnWcKMeC6FogmQf5UrCoC.ts' for reading
[tcp @ 000002271cd36c40] Starting connection attempt to 198.18.0.62 port 443
[tcp @ 000002271cd36c40] Successfully connected to 198.18.0.62 port 443
[AVIOContext @ 000002271d14c7c0] Statistics: 59024 bytes read, 0 seeks
[https @ 000002271cd10980] Opening 'https://vod-abematv.akamaized.net/ppv/chase/slot/9uYsUEkm87qk6K/180/playlist.m3u8?aver=AXmEPYkegEgNTZ&ccf=0&dt=pc_firefox&enc=clear&kg=2161' for reading
[hls @ 000002271cc6b700] Skip ('#EXT-X-VERSION:3')
[hls @ 000002271cc6b700] Skip ('#EXT-X-DISCONTINUITY-SEQUENCE:0')
[hls @ 000002271cc6b700] Skip ('#EXT-X-DISCONTINUITY')
    Last message repeated 2 times
[hls @ 000002271cc6b700] HLS request for url 'https://vod-abematv.akamaized.net/tspglive/payperview-5/9uYsUEkm87qk6K/527-580_s2_p100/h264/180/2GnWcKMeC6FvijBn96GSZq.ts', offset 0, playlist 0
[hls @ 000002271cc6b700] Opening 'crypto+https://vod-abematv.akamaized.net/tspglive/payperview-5/9uYsUEkm87qk6K/527-580_s2_p100/h264/180/2GnWcKMeC6FvijBn96GSZq.ts' for reading
[tcp @ 000002271d1261c0] Starting connection attempt to 198.18.0.62 port 443
[tcp @ 000002271d1261c0] Successfully connected to 198.18.0.62 port 443
[AVIOContext @ 000002271d23dec0] Statistics: 83088 bytes read, 0 seeks
[hls @ 000002271cc6b700] HLS request for url 'https://vod-abematv.akamaized.net/tspglive/payperview-5/9uYsUEkm87qk6K/527-580_s2_p100/h264/180/2GnWcKMeC6FfSfngx3vz1t.ts', offset 0, playlist 0
[hls @ 000002271cc6b700] Opening 'crypto+https://vod-abematv.akamaized.net/tspglive/payperview-5/9uYsUEkm87qk6K/527-580_s2_p100/h264/180/2GnWcKMeC6FfSfngx3vz1t.ts' for reading
[tcp @ 000002271d1261c0] Starting connection attempt to 198.18.0.62 port 443
[tcp @ 000002271d1261c0] Successfully connected to 198.18.0.62 port 443
[AVIOContext @ 000002271d1dfc00] Statistics: 126704 bytes read, 0 seeks
[https @ 000002271cd10980] Opening 'https://vod-abematv.akamaized.net/ppv/chase/slot/9uYsUEkm87qk6K/180/playlist.m3u8?aver=AXmEPYkegEgNTZ&ccf=0&dt=pc_firefox&enc=clear&kg=2161' for reading
[hls @ 000002271cc6b700] Skip ('#EXT-X-VERSION:3')
[hls @ 000002271cc6b700] Skip ('#EXT-X-DISCONTINUITY-SEQUENCE:0')
[hls @ 000002271cc6b700] Skip ('#EXT-X-DISCONTINUITY')
    Last message repeated 2 times
[hls @ 000002271cc6b700] HLS request for url 'https://vod-abematv.akamaized.net/tspglive/payperview-5/9uYsUEkm87qk6K/527-580_s2_p100/h264/180/2GnWcKMeC6G3kUP1HRwgQ1.ts', offset 0, playlist 0
[hls @ 000002271cc6b700] Opening 'crypto+https://vod-abematv.akamaized.net/tspglive/payperview-5/9uYsUEkm87qk6K/527-580_s2_p100/h264/180/2GnWcKMeC6G3kUP1HRwgQ1.ts' for reading
[tcp @ 000002271d1261c0] Starting connection attempt to 198.18.0.62 port 443
[tcp @ 000002271d1261c0] Successfully connected to 198.18.0.62 port 443
[AVIOContext @ 000002271d149e00] Statistics: 144944 bytes read, 0 seeks
[https @ 000002271cd10980] Opening 'https://vod-abematv.akamaized.net/ppv/chase/slot/9uYsUEkm87qk6K/180/playlist.m3u8?aver=AXmEPYkegEgNTZ&ccf=0&dt=pc_firefox&enc=clear&kg=2161' for reading
[hls @ 000002271cc6b700] Skip ('#EXT-X-VERSION:3')
[hls @ 000002271cc6b700] Skip ('#EXT-X-DISCONTINUITY-SEQUENCE:0')
[hls @ 000002271cc6b700] Skip ('#EXT-X-DISCONTINUITY')
    Last message repeated 2 times
[hls @ 000002271cc6b700] HLS request for url 'https://vod-abematv.akamaized.net/tspglive/payperview-5/9uYsUEkm87qk6K/527-580_s2_p100/h264/180/2GnWcKMeC6FnnnPMtTkYEY.ts', offset 0, playlist 0
[hls @ 000002271cc6b700] Opening 'crypto+https://vod-abematv.akamaized.net/tspglive/payperview-5/9uYsUEkm87qk6K/527-580_s2_p100/h264/180/2GnWcKMeC6FnnnPMtTkYEY.ts' for reading
[tcp @ 000002271cd3ef00] Starting connection attempt to 198.18.0.62 port 443
[tcp @ 000002271cd3ef00] Successfully connected to 198.18.0.62 port 443
[AVIOContext @ 000002271d19c6c0] Statistics: 108288 bytes read, 0 seeks
[hls @ 000002271cc6b700] HLS request for url 'https://vod-abematv.akamaized.net/tspglive/payperview-5/9uYsUEkm87qk6K/527-580_s2_p100/h264/180/7axwdV3cijdwhzqHX6KssD.ts', offset 0, playlist 0
[hls @ 000002271cc6b700] Opening 'crypto+https://vod-abematv.akamaized.net/tspglive/payperview-5/9uYsUEkm87qk6K/527-580_s2_p100/h264/180/7axwdV3cijdwhzqHX6KssD.ts' for reading
[tcp @ 000002271cd3ef00] Starting connection attempt to 198.18.0.62 port 443
[tcp @ 000002271cd3ef00] Successfully connected to 198.18.0.62 port 443
[hls @ 000002271cc6b700] Error when loading first segment 'https://vod-abematv.akamaized.net/tspglive/payperview-5/9uYsUEkm87qk6K/527-580_s2_p100/h264/180/Rs5LVsnUiFQSAeCCU7Q83W.ts'
[AVIOContext @ 000002271d405440] Statistics: 119424 bytes read, 0 seeks
[AVIOContext @ 000002271d14b1c0] Statistics: 353068 bytes read, 0 seeks
[AVIOContext @ 000002271d121740] Statistics: 87394 bytes read, 0 seeks
https://vod-abematv.akamaized.net/ppv/chase/slot/9uYsUEkm87qk6K/180/playlist.m3u8?aver=AXmEPYkegEgNTZ&ccf=0&dt=pc_firefox&enc=clear&kg=2161: Invalid data found when processing input


ERROR: ffmpeg exited with code 1
  File "yt_dlp\__main__.py", line 17, in <module>
  File "yt_dlp\__init__.py", line 984, in main
  File "yt_dlp\__init__.py", line 974, in _real_main
  File "yt_dlp\YoutubeDL.py", line 3402, in download
  File "yt_dlp\YoutubeDL.py", line 3377, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1541, in extract_info
  File "yt_dlp\YoutubeDL.py", line 1552, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1649, in __extract_info
  File "yt_dlp\YoutubeDL.py", line 1708, in process_ie_result
  File "yt_dlp\YoutubeDL.py", line 2815, in process_video_result
  File "yt_dlp\YoutubeDL.py", line 3280, in process_info
  File "yt_dlp\YoutubeDL.py", line 3003, in dl
  File "yt_dlp\downloader\common.py", line 456, in download
  File "yt_dlp\downloader\external.py", line 75, in real_download
  File "yt_dlp\YoutubeDL.py", line 1035, in report_error
  File "yt_dlp\YoutubeDL.py", line 964, in trouble```

**Comment by bashonly** [comment]
Duplicate of #7209
